sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpClient.cs

```diff
@@ -175,6 +175,29 @@ internal class AmqpClient : TransportClient
             );
         }
 
+        /// <summary>
+        ///   Creates a rule manager strongly aligned with the active protocol and transport,
+        ///   responsible for adding, removing and getting rules from the Service Bus subscription.
+        /// </summary>
+        ///
+        /// <param name="entityPath">The name of the Service Bus entity to which the rule manager is bound.</param>
+        /// <param name="retryPolicy">The policy which governs retry behavior and try timeouts.</param>
+        ///
+        /// <returns>A <see cref="TransportRuleManager"/> configured in the requested manner.</returns>
+        public override TransportRuleManager CreateRuleManager(
+            string entityPath,
+            ServiceBusRetryPolicy retryPolicy)
```

Comment:
Track 2 takes options as a parameter. Should it be the case here as well, where one of the options is the retry policy?

Reply:
We pass the options in the top level ServiceBusClient. This the amqp layer, we don't pass the options here. 

In Track-2 we pass the retryPolicy only in the top level client and use the same when we create senders, receivers, processors or rule manager, so options are not needed for [CreateRuleManager](https://github.com/ShivangiReja/azure-sdk-for-net/blob/AddRuleManager/sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs#L566).